### PR TITLE
txr: 288 -> 289

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
x86-64_darwin fails for some reason

```
./txr   tests/018/noclose.txr  > tst/tests/018/noclose.out ; case $? in ( 0 ) diff -u tests/018/noclose.expected tst/tests/018/noclose.out ;; ( 13 ) printf "SKIP %s\n" tests/018/noclose.txr ; exit 0 ;; ( * ) exit 1 ;; esac
touch tst/tests/018/noclose.ok
touch tests/010/cons.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/cons.tl  > tst/tests/010/cons.out
if ! diff -u tests/010/cons.expected tst/tests/010/cons.out ; then exit 1 ; fi
touch tst/tests/010/cons.ok
touch tests/010/hash.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/hash.tl  > tst/tests/010/hash.out
if ! diff -u tests/010/hash.expected tst/tests/010/hash.out ; then exit 1 ; fi
touch tst/tests/010/hash.ok
touch tests/010/json.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/json.tl  > tst/tests/010/json.out
if ! diff -u tests/010/json.expected tst/tests/010/json.out ; then exit 1 ; fi
touch tst/tests/010/json.ok
touch tests/010/qquote.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/qquote.tl  > tst/tests/010/qquote.out
if ! diff -u tests/010/qquote.expected tst/tests/010/qquote.out ; then exit 1 ; fi
touch tst/tests/010/qquote.ok
touch tests/010/range.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/range.tl  > tst/tests/010/range.out
./txr: unhandled exception of type type-error:
./txr: from: #<garbage: 7faa281c5ee0> is not of type range
./txr: during evaluation at tests/010/range.tl:74 of form (equal #:g0248 #:g0249)
make: *** [Makefile:472: tst/tests/010/range.ok] Error 1
rm tests/010/qquote.expected tests/010/span-var.expected tests/008/call-2.expected tests/010/json.expected tests/010/cons.expected tests/008/mdo.expected tests/010/hash.expected tests/008/no-stdin-hang.expected tests/010/range.expected
error: builder for '/nix/store/3klpgdnywrk4avy5c8nh5s09xwlkgj2v-txr-289.drv' failed with exit code 2;
```


maybe just disable tests or mark it broken.

**Comment by SuperSandro2000** [comment]
then we could just disable tests on darwin or mark it broken.

**Comment by AndersonTorres** [comment]
trealla and txr are both failing on Darwin Intel. I believe it is because of some old SDK...

**Comment by SuperSandro2000** [comment]
> trealla and txr are both failing on Darwin Intel. I believe it is because of some old SDK...

We could build them with the 11.0 SDK but if you don't want to bother with that, that's fine for me.

File: pkgs/tools/text/txr/default.nix [hunk 1]
```diff
@@ -69,5 +69,6 @@ stdenv.mkDerivation (finalAttrs: {
     license = lib.licenses.bsd2;
     maintainers = with lib.maintainers; [ AndersonTorres dtzWill ];
     platforms = lib.platforms.all;
+    broken = stdenv.isDarwin && stdenv.isx86_64;
```
Comment (SuperSandro2000) [inline pkgs/tools/text/txr/default.nix:72]: meta.broken should be accompanied with a comment

**Pull request merged**